pkg/storage/ingest/fetcher_test.go

```diff
@@ -834,76 +831,6 @@ func TestConcurrentFetchers(t *testing.T) {
 		pollFetchesAndAssertNoRecords(t, fetchers)
 	})
 
-	t.Run("staggered production with one less than multiple of concurrency and records per fetch", func(t *testing.T) {
```

Comment:
now that we don't do exact number of records this test becomes to a large extent non-deterministic. Its purpose was to test an off-by-one condition to the previous test, but achieving this with tests is really hard now because of byte estimations